### ISSUE TITLE
[Graph] Negative "axis" attribute

### ISSUE THREAD

**Issue by mciprian13** [pytorch/glow#4004]
The ONNX specifications for some of the operators allow for negative **axis** attribute: **Softmax** and **Flatten**. A negative axis value means counting dimensions from the back. Should also change the implementation of the **flattenCdr** function.

**Comment by mciprian13** [comment]
Solved in #4544. Closing this issue.
